src/MessageManager.h

```diff
@@ -92,6 +92,7 @@ struct Message
 	explicit Message( const RString &s );
 	explicit Message(const MessageID id);
 	Message( const RString &s, const LuaReference &params );
+	Message(Message&& other);
```

Comment:
Already looking into a [move constructor](http://en.cppreference.com/w/cpp/language/move_constructor)? Out of curiosity, why? I personally don't object to moving forward with more c++1y items, but I'm curious.